samples/setEndpoint.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function setEndpoint() {
+  // [START language_set_endpoint]  
```

Comment:
This is a little bit off how we usually write samples, the `[START]`/`[STOP]` should inclose an internal `async` function, which demonstrates all the functionality of this sample:

```js
function main(arg1 = 'default_value_one', arg2 = 1234) {
  // [START your_region_tag]
  /**
   * TODO(developer): Uncomment these variables before running the sample.
   */
  // const arg1 = 'default_value_one';
  // const arg2 = 1234;

  // Imports the Google Cloud Some API library
  const {SomeApiClient} = require('@google-cloud/some-api');

  // Instantiates a client
  const someApiClient = new SomeApiClient();

  async function doSomething() {
    ...
  }

  doSomething();
  // [END your_region_tag]
}

main(...process.argv.slice(2));
```

Reply:
(similar to the vision sample) The intention of the sample is to only highlight that small section of the function, the rest of the function is being left so that the tests still pass as this is a modified version of the quickstart. Sorry, should have mentioned this with the PR.

Python canonical (within the tags):
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/750d2f6d36b1a030f7226067922c8ac3dab3a39b/language/cloud-client/v1/set_endpoint.py